dart/lib/src/protocol/sentry_stack_frame.dart

```diff
@@ -0,0 +1,103 @@
+class SentryStackFrame {
+  List<String> preContext;
+  List<String> postContext;
+  Map<String, String> vars;
+  List<int> framesOmitted;
+  String filename;
+  String function;
+  String module;
+  String lineno;
+  String colno;
+  String absPath;
+  String contextLine;
+  bool inApp;
+  String _package;
+  bool _native;
```

Comment:
are they reserved words like in Java? we dont need the `_` if not

Reply:
✅